src/ds3.h

```diff
@@ -18,7 +18,6 @@
 
 #include <stdint.h>
 #include <string.h>
-#include <curl/curl.h>
```

Comment:
I believe that Rob needed it to for some error code things.  But if it compiles without it I'm wondering what happened to it.